deeppavlov/metrics/accuracy.py

```diff
@@ -106,3 +106,13 @@ def round_accuracy(y_true, y_predicted):
     examples_len = len(y_true)
     correct = sum([y1 == y2 for y1, y2 in zip(y_true, predictions)])
     return correct / examples_len if examples_len else 0
+
+
+@register_metric('kbqa_accuracy')
+def kbqa_accuracy(y_true, y_predicted):
+    total_correct = 0
+    for answer_true, answer_predicted in zip(y_true, y_predicted):
+        if answer_predicted in list(answer_true):
```

Comment:
```suggestion
        if answer_predicted in answer_true:
```
Are you sure you need casting here?